lib/src/extensions/ds_delivery_report_status.extension.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/foundation.dart';
+
+extension DSDeliveryReportStatusExtension on DSDeliveryReportStatus {
+  DSDeliveryReportStatus getValue(String value) =>
+      DSDeliveryReportStatus.values.firstWhere((e) => describeEnum(e) == value,
```

Comment:
As pointed out by the describeEnum doc, I think it's better to use the newer ```name``` getter instead of describeEnum.

Reference: https://api.flutter.dev/flutter/foundation/describeEnum.html

```suggestion
      DSDeliveryReportStatus.values.firstWhere((e) => e.name == value,
```